integr8ly/installation
# Intly 556 remove 3scale operator

## What

* Remove the use of the integreatly 3scale operator, install directly from template in ansible instead.
* Externalise all 3scale resources, installs from template in external 3scale repos.
* Remove threescale_pvc_size_mysql option, wasn't part of the official template.
* Use admin_token task exclusively for reading access token and setting up admin route facts.

## Verification

* Ensure version 2.4.0 of 3scale is installed
* Verify all expected routes are created wt2, wtc3 etc...
* Verify all expected users are cerated evalso1 etc...
* Verify auth provider is configured correctly

## Jira
https://issues.jboss.org/browse/INTLY-556

## Notes

There are deliberately 2 commits in this PR, first will install 3scale 2.2.0.GA, and the HEAD installs the current supported GA version 2.4.0.GA. This is intentional to support some testing around GA updates that we are currently working on.
